tests/Unit/ParallelAlgorithms/LinearSolver/Schwarz/Test_ComputeTags.cpp

```diff
@@ -0,0 +1,138 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "Framework/TestingFramework.hpp"
+
+#include <array>
+#include <string>
+
+#include "DataStructures/DataBox/DataBox.hpp"
+#include "DataStructures/DataVector.hpp"
+#include "DataStructures/Tensor/Tensor.hpp"
+#include "Domain/InterfaceComputeTags.hpp"
+#include "Domain/Structure/Element.hpp"
+#include "Domain/Tags.hpp"
+#include "Helpers/DataStructures/DataBox/TestHelpers.hpp"
+#include "NumericalAlgorithms/Spectral/Mesh.hpp"
+#include "NumericalAlgorithms/Spectral/Spectral.hpp"
+#include "ParallelAlgorithms/LinearSolver/Schwarz/ComputeTags.hpp"
+#include "ParallelAlgorithms/LinearSolver/Schwarz/Tags.hpp"
+
+namespace {
+struct DummyOptionsGroup {};
+}  // namespace
+
+namespace LinearSolver::Schwarz {
+
+SPECTRE_TEST_CASE("Unit.ParallelSchwarz.ComputeTags",
+                  "[Unit][ParallelAlgorithms][LinearSolver]") {
+  {
+    TestHelpers::db::test_compute_tag<
+        Tags::IntrudingExtentsCompute<2, DummyOptionsGroup>>(
+        "IntrudingExtents(DummyOptionsGroup)");
+    const auto box =
+        db::create<db::AddSimpleTags<domain::Tags::Mesh<2>,
+                                     Tags::MaxOverlap<DummyOptionsGroup>>,
+                   db::AddComputeTags<
+                       Tags::IntrudingExtentsCompute<2, DummyOptionsGroup>>>(
+            Mesh<2>{{{3, 2}},
+                    Spectral::Basis::Legendre,
+                    Spectral::Quadrature::GaussLobatto},
+            size_t{2});
```

Comment:
I don't like those literals very much, at least haven't seen a compelling reason to prefer them over `size_t{2}`. The latter is standard C++, whereas the literals are custom-defined. If you have an argument in favor of the literals, I'd be interested to hear

Reply:
I generally favor literal suffixes because it avoids possible casting at runtime, though I think the static casts for these must be compiled out (I think it does matter for floats vs doubles for instance)... and you're right that our custom literal suffix is just a static cast anyway, not redirected to the `U` or `UL` or whatnot, so I can't see this making any difference apart from the very small style preference, so I'm fine with you leaving it as you please :shrug:.